cpp/tests/copying/scatter_list_tests.cpp

```diff
@@ -951,13 +958,13 @@ TYPED_TEST(TypedScatterListsTest, NullListsOfStructs)
   auto expected_lists_null_mask_begin =
     cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i != 4; });
 
-  auto expected_lists =
-    cudf::make_lists_column(6,
-                            offsets_column{0, 3, 5, 9, 11, 11, 13}.release(),
-                            expected_structs.release(),
-                            1,
-                            cudf::test::detail::make_null_mask(expected_lists_null_mask_begin,
-                                                               expected_lists_null_mask_begin + 6));
+  std::tie(null_mask, null_count) = cudf::test::detail::make_null_mask(
```

Comment:
Ah, we `std::tie` these because they're already defined previously?

Reply:
Yes.